Release 0.2.1 (#478)
publiccodenet/standard: criteria/document-maturity.md

* Add a prominent header to every interface that indicates the maturity level of the code.
* Version all releases.

> * Especially in 'rolling release' scenarios, the version may be automatically derived from the version control system metadata (e.g. using [git describe](https://git-scm.com/docs/git-describe)).

Reviewer: ```suggestion
* Especially in 'rolling release' scenarios, the version may be automatically derived from the version control system metadata (for example by using [git describe](https://git-scm.com/docs/git-describe)).
```
